### ISSUE TITLE
[TRANSLATIONS] - Portuguese (PT-PT and PT-BR) 

### ISSUE THREAD

**Issue by paladique** [microsoft/Data-Science-For-Beginners#136]
### PT-PT
- [x] Base README.md
- [ ] Quizzes
- [ ] Introduction base README
   - [ ] Defining Data Science README
   - [ ] Defining Data Science assignment
   - [ ] Ethics README
   - [ ] Ethics assignment
   - [ ] Defining Data README
   - [ ] Defining Data assignment
   - [ ] Stats and Probability README
   - [ ] Stats and Probability assignment
- [ ] Working with Data base README
   - [ ] Relational Databases README
   - [ ] Relational Databases assignment
   - [ ] Non-Relational Data README
   - [ ] Non-Relational Data assignment
   - [ ] Python README
   - [ ] Python assignment
   - [ ] Data Preparation README
   - [ ] Data Preparation assignment
- [ ] Vizualization base README
   - [ ] Quantities README
   - [ ] Quantities assignment
   - [ ] Distributions README
   - [ ] Distributions assignment
   - [ ] Proportions README
   - [ ] Proportions assignment
   - [ ] Relationships README
   - [ ] Relationships assignment
   - [ ] Meaningful Visualizations README
   - [ ] Meaningful Visualization assignment
- [ ] Data Science Lifecycle base README
   - [ ] Introduction README
   - [ ] Introduction assignment
   - [ ] Analyzing README
   - [ ] Analyzing assignment
   - [ ] Communication README
   - [ ] Communication assignment
- [ ] Data Science in the Cloud base README
   - [ ] Intro README
   - [ ] Intro assignment
   - [ ] Low Code README
   - [ ] Low Code  assignment
   - [ ] Azure README
   - [ ] Azure  assignment
- [ ] Data Science in the Wild base README
   - [ ] Real World Examples README
   - [ ] Real World Examples assignment

### PT-BR
- [x] Base README.md
- [ ] Quizzes
- [x] Introduction base README
   - [x] Defining Data Science README
   - [x] Defining Data Science assignment
   - [x] Ethics README
   - [x] Ethics assignment
   - [x] Defining Data README
   - [x] Defining Data assignment
   - [x] Stats and Probability README
   - [x] Stats and Probability assignment
- [ ] Working with Data base README
   - [ ] Relational Databases README
   - [ ] Relational Databases assignment
   - [ ] Non-Relational Data README
   - [ ] Non-Relational Data assignment
   - [ ] Python README
   - [ ] Python assignment
   - [ ] Data Preparation README
   - [ ] Data Preparation assignment
- [x] Vizualization base README
   - [x] Quantities README
   - [x] Quantities assignment
   - [x] Distributions README
   - [x] Distributions assignment
   - [ ] Proportions README
   - [ ] Proportions assignment
   - [ ] Relationships README
   - [ ] Relationships assignment
   - [ ] Meaningful Visualizations README
   - [ ] Meaningful Visualization assignment
- [x] Data Science Lifecycle base README
   - [ ] Introduction README
   - [ ] Introduction assignment
   - [ ] Analyzing README
   - [ ] Analyzing assignment
   - [ ] Communication README
   - [ ] Communication assignment
- [ ] Data Science in the Cloud base README
   - [ ] Intro README
   - [ ] Intro assignment
   - [ ] Low Code README
   - [ ] Low Code  assignment
   - [ ] Azure README
   - [ ] Azure  assignment
- [ ] Data Science in the Wild base README
   - [ ] Real World Examples README
   - [ ] Real World Examples assignment

**Comment by Nockkk** [comment]
Hello,
I am Portuguese. If you want, I can work on this issue. I can translate to pt-pt.

**Comment by izzy-el** [comment]
Hey, I am from Brazil. Would it be interested to translate the files to PT-BR (Brazilian Portuguese) as well?

**Comment by paladique** [comment]
@Nockkk @izzy-el absolutely! I have assigned the issue to you both. I will let you both decide on which parts you'd prefer to translate. Thank you!

**Comment by izzy-el** [comment]
Hey @paladique. We are from different countries so, even though we both speak Portuguese, there are some differences in the language. So we decided to translate, to both PT-BR (Brazil) and PT-PT (Portugal). Is that okay?

**Comment by fernandakawasaki** [comment]
Hi, I would like to help translate to pt-br too! What parts are you working on @izzy-el ? Just to make sure I pick something different.

**Comment by izzy-el** [comment]
Hey @fernandakawasaki , great! So far I have finished the base README and currently I am working on the Introduction READMEs. I should open a draft PR soon.

**Comment by fernandakawasaki** [comment]
Ok! I will translate the Visualization base section

**Comment by Midas-sudo** [comment]
@paladique  I would like to help with the pt-PT translation, could you assist me to the open issue as well?

**Comment by paladique** [comment]
@Midas-sudo sure! You're all set :)

**Comment by souzaspa** [comment]
Good Morning, 

Hello,
I am brazilian and if you want, I can translate parts on this issue. 
Thanks.

**Comment by annezao** [comment]
cc: @paladique 
Hey everyone, I'm willing to help on translating to pt-br. It seems like cap 5 and 6 need translations.

**Comment by m6urns** [comment]
I'd be willing to help with translating pt-br, I regularly translate engineering documentation into Portuguese and would be happy to help with some chapters here.

**Comment by leestott** [comment]
Translations are now automatically completed using coop translator
